tycho-surefire/tycho-surefire-plugin/src/main/java/org/eclipse/tycho/surefire/AbstractEclipseTestMojo.java

```diff
@@ -528,6 +528,18 @@ public abstract class AbstractEclipseTestMojo extends AbstractTestMojo {
     @Parameter
     private int reactorConcurrencyLevel;
 
+    public enum ClassLoaderOrder {
```

Comment:
It would be nice if this enum didn't have to be duplicated.

Reply:
@joeshannon I would just use a string in `OsgiSurefireBooter`. Imo, we can assume the value is valid once it gets there.

Reply:
Yes no need for an enum on the booter level, as this is quite low-level.

Reply:
Also, you already throw an exception if it's not in the valid range of values.

Reply:
Yes that makes sense, have updated.